community/concepts/deep-then-wide.md

```diff
@@ -0,0 +1,51 @@
+# deep then wide
+
+SourceCred is still a young project. As we grow it, we have a choice of priorities:
+
+ - do we want to go "wide" (adoption by lots of projects) or
+ - go "deep" (intensive usage by a few projects)?
+
+#### What does it mean to go wide?
+
+Going "wide" means encouraging lots of communities to start using SourceCred.
+Since SourceCred can run on any GitHub repository or Discourse forum, we could
+roll cred out to many different projects and groups. We could optimize for the
+number of projects using SourceCred, and focus on features that make the scores
+more salient or engaging — for example, making it easy to generate "weekly
+reports" showing the new cred being earned by contributors.
+
+Going wide is tempting because progress is very easy to measure, and the project
+would get plenty of buzz which could bring more contributors and opportunities.
+
+#### What does it mean to go deep?
+
+Going deep means that instead of focusing on finding many projects that will
+use SourceCred, we'll instead focus on making SourceCred work incredibly well
+for a few communities. In part, this means paying a lot of attention to the edge
+cases, and spots where SourceCred isn't working as intended, and trying to fix
+those issues.
+
+In many ways, going deep is more challenging than going wide. We could go wide
```

Comment:
>We could go wide by marketing SourceCred and building buzz, but improving the core quality requires
a lot of experimentation with algorithms and community practices.

When you use 'but' in this way, the part of the sentence after 'but' should contradict the part before the but. That isn't happening ere, which is confusing... As we already talk about the temptation of creating buzz in the previous section, I think we can delete this para.

Reply:
Removed.